server/api_server/logs_endpoint.py

```
@@ -0,0 +1,58 @@
+    log_path = os.path.join(logPath, log_file)
+
+    try:
+        # Purge content
+        with open(log_path, "w") as f:
+            f.write("File manually purged\n")
+        msg = f"[clean_log] File {log_file} purged successfully"
+
+        mylog('minimal', [msg])
+        write_notification(msg, 'interrupt')
+        return jsonify({"success": True, "message": msg}), 200
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)